### PR TITLE
filter layouts; colour highlighting of search results

### PR DESCRIPTION
**Pull request by dholler1011**
Add filter to backend layouts; colour highlighting of search results in backend layouts and backend layersets
see internal ticket 8123

### CODE REVIEW

**Comment by astroidex** [comment]
- works fine
- let's discuss the color in the Mapbender team. Maybe .bg-info-subtle would fit better
- https://getbootstrap.com/docs/5.3/utilities/background/

**Review by Phocacius — CHANGES_REQUESTED** [review]
All in all good job, I have two minor remarks.

File: src/Mapbender/ManagerBundle/Resources/views/Application/form-elements.html.twig [hunk 1]
```diff
@@ -29,11 +29,21 @@
   {% endblock %}
 {%- endif -%}
 
+<div class="row">
```
[the remark this answers is not in the document]
Comment (dholler1011) [reply src/Mapbender/ManagerBundle/Resources/views/Application/form-elements.html.twig:32]: Sorry, I don't know what you exactly mean. Can you please explain this in more detail?

[the remark this answers is not in the document]
Comment (Phocacius) [reply src/Mapbender/ManagerBundle/Resources/views/Application/form-elements.html.twig:32]: Sorry for being unclear here. In the layout box with the filter there is a significant margin at the top of the box. When comparing to the layerset filter you can see it clearly:

Layout:

grafik

Layerset:

grafik

**Comment by astroidex** [comment]
- I tested the feature. It looks good.

Still some fixes nedded

1. avoid filter for WMS or WMTS in list of WMS-Instances
- because it will highlight all services
- if it is too much effort we can ignore it
photo_2024-10-17_13-46-48


2. Filter in Sources / Shared instance list -  Allow to search for the Title and WMS number
- at the moment you can not search for a word in the title of the WMS or for the number of the WMS
- this should be possible

**Comment by Phocacius** [comment]
My previous comments are fixed, thank you! Code looks all good now,

I have one additional remark to Astrids comments: Searching for the placeholder dummy text should also not be possible:

grafik

**Review by Phocacius — APPROVED** [review]

**Pull request merged**